sass-css.md

```diff
@@ -1,33 +1,57 @@
-SASS/CSS is _not_ code, but you can apply a lot the same principles to it. Knowing how it's different though, and different again on a large site, is key to producing reusable, fast and understandable styling.
+# [HouseTrip](http://www.housetrip.com) SASS-CSS Style Guide {
+
+## Preamble: the TL,DR
 
 On a large site the most important thing to understand is to strive for re-use as much as possible.
 
 With that end the overall strategy of our SASS/CSS is to take some well understood (and battle-tested) functionality from bootstrap and add our components on top.
 
-> With great power, comes great responsibility.
-> --Uncle Ben (the Spiderman one, not the rice one)
+- We prefer the indented SASS syntax
```

Comment:
We do? Snuck that one through @mrmohtas... I don't really care, but ideally we shouldn't be indenting at all if we can avoid it.